### PR TITLE
Extend zoom excmd keybindings and improve documentation.

### PR DESCRIPTION
**Pull request by antonva**
This patch improves upon the existing zoom excmd and adds some features present
in Vimperator and Pentadactyl. `zI` and `zO` are now bound to the maximum and
minimum zoom levels respectively. `zM` (read zoom more) zooms in on increments
of 50pp and `zR` (read zoom reduce) zooms out on increments of 50pp.

In addition, overshooting the zoom level while using relative adjustments is now
handled by setting the level to it's max/min zoom level.

An error is thrown if an absolute value is given that is out of bounds of the
allowed zoom range.

### CODE REVIEW

File: src/lib/config.ts [hunk 1]
```diff
@@ -250,7 +250,11 @@ class default_config {
         A: "bmark",
         zi: "zoom 0.1 true",
         zo: "zoom -0.1 true",
```
Comment (bovine3dom) [inline src/lib/config.ts]: dupe?

File: src/lib/config.ts [hunk 2]
```diff
@@ -405,6 +409,7 @@ class default_config {
             "fillcmdline_tmp 3000 !jsb is deprecated. Please use jsb instead",
         current_url: "composite get_current_url | fillcmdline_notrail ",
         stop: "js window.stop()",
+        zo: "zoom",
```
Comment (bovine3dom) [inline src/lib/config.ts]: dupe?

File: src/excmds.ts [hunk 3]
```diff
@@ -1797,10 +1797,28 @@ export function geturlsforlinks(reltype = "rel", rel: string) {
     return []
 }
 
+/** Sets the current page's zoom level anywhere between 30% and 300%.
+ *
+ * If you overshoot the level while using relative adjustments i.e. level > 300% or level < 30%
+ * the zoom level will be set to it's maximum or minimum position.
+ *
+ * @param level - The zoom level to set.
+ * Expects percentages when changing the absolute zoom value and percentage points when making relative adjustments..
```
Comment (bovine3dom) [inline src/excmds.ts]: too many dots

File: src/lib/config.ts (hunk 1 above)
Comment (antonva) [reply src/lib/config.ts]: One is shorthand on the commandline `:zo[om]` the other is a keybind.

File: src/excmds.ts (hunk 3 above)
Comment (antonva) [reply src/excmds.ts]: Gah, blaming my keyboard for that.

File: src/lib/config.ts (hunk 1 above)
Comment (antonva) [reply src/lib/config.ts]: As in, one is an exalias the other a keybind.

Comment (bovine3dom) [reply src/lib/config.ts]: oh wow. ta.

**Pull request merged**